### ISSUE TITLE
[expo-image]: Prop "resizeMode" is deprecated, use "contentFit" instead

### ISSUE THREAD

**Issue by philiplindberg** [nandorojo/solito#366]
I'm using `SolitoImage` in my react native app, but I'm seeing the following warning in the terminal whenever an image is rendered on ios and android:

```
[expo-image]: Prop "resizeMode" is deprecated, use "contentFit" instead
```

Here's a repro that demonstrates the issue: https://github.com/philiplindberg/solito-repro

**Comment by deggertsen** [comment]
I submitted a pull request for this here: https://github.com/nandorojo/solito/pull/371

**Comment by nandorojo** [comment]
Published in 3.2.2
